docs/topics/OBST-wf-swarm-crud-introduction.adoc

```diff
@@ -0,0 +1,28 @@
+= Introduction
```

Comment:
This has a huge overlap with the other intros, you should unify this to a single topic and use it as a common include.